### PR TITLE
Remote control - display the authorization dialog in meet

### PR DESCRIPTION
**Pull request by hristoterezov**
Jitsi Meet part of the PR - https://github.com/jitsi/jitsi-meet/pull/1541

Depends on - https://github.com/jitsi/jitsi-meet-electron/pull/9

### CODE REVIEW

File: modules/remotecontrol/index.js [hunk 1]
```diff
@@ -173,16 +181,16 @@ class RemoteControl {
...
-                //Open Dialog and answer
-                this.handlePermissionRequest({
-                    userId: event.userId,
-                    userJID: event.userJID,
-                    displayName: event.displayName,
-                    screenSharing: event.screenSharing
-                });
+                if(event.action === PERMISSIONS_ACTIONS.request
+                    && this.handleAuthorization) {
+                    //Open Dialog and answer
```
Comment (saghul) [inline modules/remotecontrol/index.js]: style: space after `//`

File: config.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 module.exports = {
-    jitsiMeetURL: "https://meet.jit.si/"
+    jitsiMeetURL: "https://meet.jit.si/",
+    handleAuthorization: false
```
Comment (saghul) [inline config.js]: Question: why would someone not want authorization to be handled? Also, why is it false by default?

Comment (hristoterezov) [inline config.js]: After some discussions recently it was decided that Jitsi Meet's default behavior will be to handle the authorization itself. This PR just follows Jitsi Meet's behavior and makes the jitsi-meet-electron project compatible with the latest versions of meet.

File: modules/remotecontrol/index.js (hunk 1 above)
Comment (hristoterezov) [inline modules/remotecontrol/index.js]: Fixed!

**Pull request merged**